packages/account/src/Sections/Security/ApiToken/__tests__/api-token.spec.tsx

```diff
@@ -1,39 +1,43 @@
 import React from 'react';
 import { FormikValues } from 'formik';
-import { act, render, screen, waitFor } from '@testing-library/react';
+import { act, fireEvent, render, screen, waitFor } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
-import { isMobile, getPropertyValue, useIsMounted, WS } from '@deriv/shared';
+import { isMobile, getPropertyValue } from '@deriv/shared';
```

Comment:
The component uses is_mobile from ui-store.right? why do we need to import isMobile from shared in its test file?

Reply:
Some of the shared deriv components used still use isMobile. So we have to mock isMobile to for mobile testing